src/main/java/me/ars/pokerbot/poker/Table.java

```diff
@@ -396,20 +396,22 @@ private void sendStatus(String turn) {
   }
 
   private void collectForcedBets() {
-    if (forcedBetType == Constants.FORCED_BET_ANTE) {
-      callback.collectAnte(Constants.ANTE);
+    if (config.ante != null) {
+      callback.collectAnte(config.ante);
 
       for (Player player : players) {
-        mainPot.collectAnte(player, Constants.ANTE);
+        mainPot.collectAnte(player, config.ante);
       }
-    } else if (forcedBetType == Constants.FORCED_BET_BLINDS) {
+    }
+
+    if (config.smallBlind != null && config.bigBlind != null) {
       final int blindPlayer = turnIndex;
       final Player smallBlindPlayer = players.get(turnIndex);
-      final int smallBlind = mainPot.collectSmallBlind(smallBlindPlayer, Constants.BIG_BLIND_AMOUNT);
+      final int smallBlind = mainPot.collectSmallBlind(smallBlindPlayer, config.bigBlind);
       lastIndex = lastUnfolded(turnIndex - 1);
       turnIndex = wrappedIncrement(turnIndex);
       final Player bigBlindPlayer = players.get(wrappedIncrement(blindPlayer));
-      final int bigBlind = mainPot.collectBigBlind(bigBlindPlayer, Constants.BIG_BLIND_AMOUNT);
+      final int bigBlind = mainPot.collectBigBlind(bigBlindPlayer, config.bigBlind);
       lastIndex = lastUnfolded(turnIndex - 1);
       turnIndex = wrappedIncrement(turnIndex);
       callback.collectBlinds(bigBlindPlayer.getName(), bigBlind, smallBlindPlayer.getName(), smallBlind);
```

Comment:
This is not really a pure refactor becuase it changes the logic a little bit. It think it's correct to allow both antes and blinds to be used at the same time, so this means one option less i the config file.

Reply:
I agree, it makes more sense this way.